### ISSUE TITLE
Compatibility with scylla manager 3.0

### ISSUE THREAD

**Issue by mrlexor** [scylladb/scylla-monitoring#1847]
**System information**
- Scylla version (you are using):
Scylladb version 5.0.5 and scylla-manager 3.0.2
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**
In order to properly working of `Scylla Manager Metrics` Grafana dashboard with scylla-manager version 3.0 need to update prometheus recording rules [here](https://github.com/scylladb/scylla-monitoring/blob/scylla-monitoring-4.1.0/prometheus/prom_rules/prometheus.rules.yml) and Grafana dashboard [here](https://github.com/scylladb/scylla-monitoring/blob/scylla-monitoring-4.1.0/grafana/scylla-manager.template.json).

According on the issue in [scylladb/scylla-manager](https://github.com/scylladb/scylla-manager/issues/3282) I found out that metrics `scylla_manager_task_*` don't exist anymore and they should be replaced on the following:

```
There were changes in SM metrics (2.6 -> 3.0) which replaced scylla_manager_task_* with scylla_manager_scheduler_*. Currently it looks like that:

scylla_manager_task_run_total -> scylla_manager_scheduler_run_total
scylla_manager_task_last_success -> scylla_manager_scheduler_last_success
scylla_manager_task_active_count -> scylla_manager_scheduler_run_indicator !!!
scylla_manager_task_last_run_duration_seconds -> no metric for that !!!
```
**Who will benefit with this feature?**
Anyone who uses the `Scylla Manager Metrics` Grafana dashboard with scylla-manager version 3.0 and higher.

**Any Other info.**
I have tested the new prometheus recording rules on our installation and I have performed some changes in `Scylla Manager Metrics` Grafana dashboard so I can say that the last version of `scylla-monitoring-4.1.0` can work properly with scylla-manager version 3.0 after the little changes.

**Comment by amnonh** [comment]
@mrlexor can you please add what you've changed, or, send a PR with the changes?

**Comment by mrlexor** [comment]
Sure, [here](https://github.com/scylladb/scylla-monitoring/pull/1848) I have created the pull request.

**Comment by amnonh** [comment]
Closed with #1848
